docs/application/vscode-ext/web.md

```diff
@@ -270,41 +257,45 @@ You can use these features to easily develop application using TAU.
 
 ### Features
 
-### Code completion
-  
-  Display a list of recommended values when specifying attributes or APIs.
+### Code Completion
+
+  Displays a list of recommended values when specifying attributes or APIs.
 
   ![Code completion](media/tizen_web20.gif)
 
-- Hover
+### Hover
   
-  Display a description of the attribute or API when the mouse over attritubes
+  Displays a description of an attribute or an API when you hover the mouse over that attribute or API.
 
-    ![Hover](media/tizen_web21.gif)
+  ![Hover](media/tizen_web21.gif)
 
-- Signature Help
+### Signature Help
   
-  Displays an information of API parameters when using the API
+  Displays information about the API parameters.
 
-    ![Signature Help](media/tizen_web22.gif)
+  ![Signature Help](media/tizen_web22.gif)
 
-- Document Viewer
+### Document Viewer
   
   Displays a document related with attributes or APIs.
-    > **How to use the Document Viewer**
-    > 
-    > Click the "Search in TAU document" on the context menu
 
-    ![Document Viewer](media/tizen_web23.gif)
+  > **How to use the Document Viewer**
+  >
+  > Place the mouse cursor on the **TAU element** you want to check and right click.
+  >
+  > Click **Search in TAU document**.
+
+  ![Document Viewer](media/tizen_web23.gif)
 
 ## Troubleshoot
 
 This section contains instructions for common problems with the Tizen Baseline Web IDE.
 
-- To troubleshoot red screen error, follow these steps:
+- Red Screen
 
-  a. Launch **Web Simulator**. 
-  b. When the red screen is created, press **F12**.  
-  c. Click **Application Tab** at the top-end immediately after the Chrome Devtool opens. 
-  d. Select **Clear storage** and click **Clear site data**. 
-  e. Exit Devtool and Simulator and launch again.
+1. In VS Code, click **View** > **Command Palette**.
+2. In the input field of the **Command Palette** that appears, type **Run Web Simulator**, and press **Enter**.
+3. If the red screen appears, press **F12**.
+4. In the **DevTools** window that appears, click **Application**. (Please provide an image if possible)
```

Comment:
Please remove the following sentence:

(Please provide an image if possible)

Reply:
done.